tools/wpt/run.py

```diff
@@ -295,10 +310,7 @@ def main():
     venv.install_requirements(os.path.join(wpt_root, "tools", "wptrunner", "requirements.txt"))
     venv.install("requests")
```

Comment:
Not necessary, but it allows the file to be run directly from the command line, which seems somewhat useful to keep for now.